dashboard/src/components/utils/LengthyOperationProgressBar.vue

```diff
@@ -14,7 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 <template>
-  <b-progress :max="max" :height="height" :variant="variant">
+  <b-progress :max="max" :height="height" :variant="variant" :animated="animated">
     <b-progress-bar :value="current" :aria-labelledby="`${name} Progress`"></b-progress-bar>
```

Comment:
I believe aria-labelledby needs to reference the element id of a label, aria-label may be what is intended here

Reply:
:+1: